samples/sqltransport-sqlpersistence/SqlServer_3/Sender/AssemblyInfo.cs

```diff
@@ -0,0 +1,9 @@
+﻿using NServiceBus.Persistence.Sql;
+
+#region SqlPersistenceSettings
+
+[assembly: SqlPersistenceSettings(
```

Comment:
Thanks @SeanFeldman - totally copied that from another sample that was trying to "show everything" - since this is a SQL Transport sample definitely only want MSSQL scripts. Since the attribute's constructor defaults all missing attributes to false, I took out the mySql bit and it still only generates MSSQL scripts.